File: src/main/java/easy/strings/Anagram.java
package easy.strings;

import java.util.Arrays;

public class Anagram {
    public boolean isAnagram(String s, String t) {
        char[] sArr = s.toCharArray();
        char[] tArr = t.toCharArray();
        Arrays.sort(sArr);
        Arrays.sort(tArr);


        if (Arrays.equals(sArr, tArr)){
            return true;
        }
        return false;
    }
}

File: src/main/kotlin/leetcode/easy/strings/Anagram.kt
package leetcode.easy.strings

class Anagram {
}

fun isAnagram(s: String, t: String): Boolean {
    val arr1 = s.toCharArray().sorted()
    val arr2 = t.toCharArray().sorted()

    if (arr1 == arr2)
        return true
    return false
}